src/playwright/playwright.config.ts

```diff
@@ -150,11 +152,10 @@ export default defineConfig({
       } satisfies CoverageReporterOptions,
     ],
   ],
-  retries: process.env.CI ? 1 : 0,
   use: {
     baseURL: 'http://localhost:6006/iframe.html',
     testIdAttribute: 'data-test',
-    trace: 'on-first-retry',
+    trace: 'retain-on-failure',
```

Comment:
Tracing on first retry has a couple problems:

- It's not obvious that you have to click the Retry tab in the test report to get a link to the trace.
- If first try fails but the retry succeeds, then the retry's trace won't be very useful because it'll be a trace of a passing test.

To solve both of the those problems, we simply create a trace whenever a test fails and don't retry.